.travis.yml

```diff
@@ -14,9 +13,9 @@ matrix:
   include:
     - php: 7.1
       env: PHPUNIT_TEST=core,cwpcore,admin,cwpcms,Default,cwpsearch PHPCS_TEST=1
-    - php: 7.2
```

Comment:
We should keep testing for `7.2`. It is still supported. Maybe we could swap `7.1` to `7.2`?

Reply:
Added it back in